### PR TITLE
Add cookbook entry for Gaussian Naive Bayes example

### PR DESCRIPTION
**Pull request by tbskm**
Add the .sg and .rst files for the cookbook entry, and remove all language-specific files for the example.

### CODE REVIEW

**Comment by karlnapf** [comment]
No need to include the translates files.
Also, no need to store the feature matrix. Only the output labels in fact. Accuracy also not needed,in the other example they were for testing

**Comment by tbskm** [comment]
Ok, thanks for the help. I'll look into the issues with Travis as well. Sorry about that!

**Comment by lisitsyn** [comment]
Looks good

**Comment by karlnapf** [comment]
Thanks a lot for this addition.
We are currently merging the sphinxdoc feature branch. Once it is done, you can re-send the PR against develop, and we will be able to look at a preview. Until then, be a bit patient :)

File: doc/cookbook/source/examples/classifier/gaussiannaivebayes.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+===================================
```
Comment (karlnapf) [inline doc/cookbook/source/examples/classifier/gaussiannaivebayes.rst]: can you rename the file to gaussian_naive_bayes.rst?

**Comment by karlnapf** [comment]
@vigsterkr Ill leave handling this to you. Can be merged from my side

**Comment by vigsterkr** [comment]
this'll need some love because i've just rebased the sphinx branch...

**Comment by karlnapf** [comment]
Please re-send this one against the develop branch. The cookbook feature branch is now merged

**Pull request closed without merging**